src/tools/bin/radiationSyntheticDetector

```diff
@@ -83,11 +83,11 @@ for myfile in args.path2Data:
     theta_min_data = args.dataExtend[2]
     theta_max_data = args.dataExtend[3]
 
-    # set extent analyzer:
-    omega_min_analy = args.analyzerExtend[0]
-    omega_max_analy = args.analyzerExtend[1]
-    theta_min_analy = args.analyzerExtend[2]
-    theta_max_analy = args.analyzerExtend[3]
+    # set extent plugin:
+    omega_min_analy = args.pluginExtend[0]
+    omega_max_analy = args.pluginExtend[1]
+    theta_min_analy = args.pluginExtend[2]
+    theta_max_analy = args.pluginExtend[3]
```

Comment:
Sorry - I was in the wrong code - here it is a **statistical analysis** this the name is correct (ROI, etc. would be confusing). 
Just leave it as it is.